kobo/apps/stripe/views.py

```diff
@@ -1,20 +1,136 @@
+import stripe
+
 from django.conf import settings
-from djstripe.models import Product, Subscription
-from rest_framework import mixins, renderers, viewsets
+from django.core.exceptions import ObjectDoesNotExist
+from djstripe.models import Customer, Product, Subscription, Price
+from djstripe.settings import djstripe_settings
+from organizations.utils import create_organization
+from rest_framework import mixins, status, viewsets
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
+from rest_framework.views import APIView
 
 from kobo.apps.stripe.serializers import (
-    SubscriptionSerializer
+    SubscriptionSerializer,
+    CheckoutLinkSerializer,
+    CustomerPortalSerializer
+)
+
+from kobo.apps.organizations.models import (
+    OrganizationUser,
+    Organization
 )
 
 
+class CheckoutLinkView(
+    APIView
+):
+    permission_classes = (IsAuthenticated,)
+    serializer_class = CheckoutLinkSerializer
+
+    @staticmethod
+    def generate_payment_link(price_id, user, organization_uid):
+        try:
+            organization = Organization.objects.get(uid=organization_uid)
```

Comment:
Use a get_or_create

Reply:
We discussed making org uid Nullable - meaning the backend should create a new org